src/SprykerEco/Yves/Computop/ComputopFactory.php

```diff
@@ -204,6 +217,14 @@ public function createEasyCreditFormDataProvider()
         return new EasyCreditFormDataProvider($this->getQuoteClient(), $this->createOrderEasyCreditMapper());
     }
 
+    /**
+     * @return \SprykerEco\Yves\Computop\Form\DataProvider\PayuCeeSingleFormDataProvider
+     */
+    public function createPayUCeeSingleFormDataProvider(): PayuCeeSingleFormDataProvider
```

Comment:
* Rename the method to `createPayuCeeSingleFormDataProvider()`.

Reply:
++